### ISSUE TITLE
Normalize capitalization of editor titles when migrating editing credits

### ISSUE THREAD

**Issue by LegalizeAdulthood** [GrandComicsDatabase/gcd-django#544]
A reviewer left the comment that when attributing editor roles to a credit, the role is written without capitalization, e.g. "editor" not "Editor".  When migrating editing credits, the software should fix the capitalization for me.

**Comment by jochengcd** [comment]
Not really feasible. The capitalization depends on the language. And since indexes might be in a foreign language, we cannot go by the language of the comic.

**Comment by LegalizeAdulthood** [comment]
Surely we can recognize the common case for English?

**Comment by jochengcd** [comment]
But we don't know when to use English ?

**Comment by LegalizeAdulthood** [comment]
If we don't know an issue is English, then how does the feature migration query work in the dropdown to recognize foreign language features?  To me that implies that something knows this issue is in a particular language when I'm editing it.

**Comment by jochengcd** [comment]
We know the language of the series, but we don't know in which language the edits are performed.

**Comment by jochengcd** [comment]
What would be a way is to have a button on the issue editing page to lowercase the entry on indexer request.

**Comment by LegalizeAdulthood** [comment]
> What would be a way is to have a button on the issue editing page to lowercase the entry on indexer request.

Yeah, this could easily be done on the front-end with some javascript.  I'm going to be setting up a development environment soon and this would be an easy change to the template.
